test/clj/harja/testi.clj

```diff
@@ -261,15 +261,20 @@
      (if (= n-kierros n)
        (throw (Exception. "Queryn yrittäminen epäonnistui"))
        (let [tulos (try+
+                      (when (pos? n-kierros)
```

Comment:
Mietin tuon pos? käyttöä, kun ymmärrän itse koodin niin, että n-kierros ei voi olla negatiivinen. ja pos? on true jos n-kierros on suurempi kuin nolla.  eli logia ei siinä tapauksessa tulisi ekalla kerralla, mutta muilla kerroilla tulisi? 
Jos tämä on se tavoite, niin olisiko helpompi vaihtaa (pos? n-kierros) vain muotoon:
(> n-kierros 0)

Reply:
Mun silmään pos? on vaan yksinkertaisempi tapa kirjoittaa tuo > 0 ja joudun aina (> ...) notaation kanssa hidastamaan ja varmistamaan että operandit menee oikein päin kun ei ole infix-notaatio :) 

Mutta muutin nyt tolleen koska voin uskoa että sun näkökanta saattaa olla yleisempi.